__tests__/db.test.ts

```diff
@@ -10,14 +10,14 @@ jest.mock("mongoose", () => ({
 }));
 
 describe("connectDB", () => {
-  let mockExit: jest.SpyInstance<void, [code?: number | undefined]>;
+  let mockExit: jest.SpyInstance<void, [code?: string | number | undefined | null]>;
```

Comment:
There a way to consolidate the type for `code`

Reply:
I think this was resolved? Removing all the typing allowed it to infer the proper type for "code"